vignettes/comparison_with_malariasimulation.Rmd

```diff
@@ -0,0 +1,193 @@
+---
+title: "Comparison with malariasimulation"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{comparison_with_malariasimulation}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+`malariasimple` is designed to resemble `malariasimulation` as closely as possible. However, there are certain scenarios (e.g. fast, low resolution runs; low population/transmission scenarios) where `malariasimple` may not provide a useful approximation. If in doubt, it is a good idea to check your `malariasimple` simulation against an equivalent `malariasimulation` run. In this vignette we will demonstrate some examples of comparing equivalent simulations of `malariasimulation` with both deterministic and stochastic runs of `malariasimple`.
+
+Note: `malariasimple` is designed to be familiar to users of `malariasimulation` and hence many functions have the same name. Hence, when both libraries are loaded in an R session simultaneously, it becomes necessary to state which version of a function you are calling by using the :: operator. 
+
+```{r, eval = FALSE}
+library(malariasimple)
+library(malariasimulation)
```

Comment:
I believe packages used in Vingettes need to be included in "Suggests" in the description (interesting this is passing checks as it is).